pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "navigator_backend"
-version = "1.12.4"
+version = "1.12.5"
```

Comment:
Is this required or do we not have auto tagging?

Reply:
It seems currently, yes, because the pre commit validator will fail otherwise:
https://github.com/climatepolicyradar/check-pyproject-package-version/blob/main/action.yml#L50-L51
Called by:
https://github.com/climatepolicyradar/reusable-workflows/blob/a43ba21a96bfdb1c6ee05283a9a8e7b50edd63b1/.github/workflows/python-precommit-validator.yml#L31-L32
Which is called by:
https://github.com/climatepolicyradar/navigator-backend/blob/main/.github/workflows/ci-cd.yml#L36